src/test/java/org/dataloader/DataLoaderValueCacheTest.java

```diff
@@ -33,7 +33,7 @@
 public class DataLoaderValueCacheTest {
 
     @ParameterizedTest
-    @MethodSource("org.dataloader.fixtures.parameterized.TestDataLoaderFactories#getWithoutPublisher")
+    @MethodSource("org.dataloader.fixtures.parameterized.TestDataLoaderFactories#get")
```

Comment:
This is where we "add" the new tests - we are now testing this against `PublisherDataLoaderFactory` and `MappedPublisherDataLoaderFactory`.

Reply:
OK I get it now - we have tests for non publisher data loaders and publisher data loaders.  So its working for the old and the new